### ISSUE TITLE
High Contrast not achieved for debugging variables 

### ISSUE THREAD

**Issue by BurtHarris** [Microsoft/vscode#28014]
- VSCode Version: 1.12.2
- OS Version: Microsoft Windows [Version 10.0.15063]

Steps to Reproduce:

1.  Select the "high contrast" color scheme
2.  Start debugging a node.js app, inserting a breakpoint somewhere interesting
3.  Point at an object variable

Result:

Insufficient contrast for members of type bool (blue), string (red) and number (green) on dark-grey background.   Medium-grey objects could benefit from being brighter.

![badcontrast](https://cloud.githubusercontent.com/assets/2715647/26768073/136f4d80-495a-11e7-86eb-aead7168b2a1.png)

**Comment by BurtHarris** [comment]
Also bad in the variables window...
![untitled](https://cloud.githubusercontent.com/assets/2715647/26768196/09daa07a-495b-11e7-87e2-4663873a1735.png)
haps related.

**Comment by BurtHarris** [comment]
I appreciate Microsoft's efforts to accommodate those of us with aging eyeballs, it would be nice if the default "High Contrast" theme were improved.   For now, I'm switching to "Abyss".

**Comment by isidorn** [comment]
Hi @BurtHarris thanks a lot for feedback. The best way to improve the high contrast theme is to give feedback and we will address those issues as soon as possible.

I have played with the colors you mentioned and here is the result:
You can try it out in the insiders channel in a couple of days.
Please create more issues if you think high contrast can be further improved.

![screen shot 2017-06-06 at 15 42 25](https://cloud.githubusercontent.com/assets/1926584/26832141/cf539c48-4ace-11e7-925a-aa2630081713.png)

**Comment by BurtHarris** [comment]
Thanks @isidorn , looking forward to trying it out.   

Is there some way to check when this commit has been merged into the insider's track and released?   My current install says 'Version 1.13.0-insider'.   

P.S. I don't see tags for insider builds at https://github.com/Microsoft/vscode/releases

**Comment by isidorn** [comment]
@BurtHarris we always release insiders every day so it means that a fix can always be picked up one day later.
However this couple of days we are in special rhythm since we are about to release a new stable build, so our insiders candidate is not picking up everything from master. I think we should be back to our normal rhythm on Friday so the fix will be picked up then.

Thanks
